fault/verilator_target.py

```diff
@@ -9,9 +158,9 @@ def __init__(self, circuit, test_vectors, directory="build/", flags=[]):
         self._flags = flags
 
     def run(self):
-        magma.testing.verilator.compile(
```

Comment:
Also eventually: should some check to ensure the verilog file is here. If not we should compile `circuit` to verilog.